R/fetch_table.R

```diff
@@ -58,10 +62,28 @@ fetch_table = function(table, query_string = NULL) {
     }
   )
 
-  res %>%
+  res_data = res %>%
     resp_body_string() %>%
     read_csv(show_col_types = FALSE) %>%
     # Due to messy data read_csv fails to assign column types.
     # To avoid polluting the console, warnings are suppressed.
     suppressWarnings()
+
+  if (pretty_colnames) {
+    switch(
+      table,
+      "ps" = {
+        res_data = res_data %>%
+          rename(any_of(setNames(names(col_labels[["ps"]]), col_labels[["ps"]])))
+      },
+      "pscomppars" = {
+        res_data = res_data %>%
+          rename(any_of(setNames(names(col_labels[["pscomppars"]]), col_labels[["pscomppars"]])))
+      },
+      warning(paste0("Table `", table, "` doesn't currently support pretty names.
+        Database column names provided instead."))
+    )
+  }
```

Comment:
issue: Why not just go with the better solution that I have provided.
```suggestion
  if (pretty_colnames) {
    labels = get(col_labels[[table]])
    if (is.null(labels)) {
     warning(paste0(
       "Table `", table, "` doesn't currently support pretty names.
        Database column names provided instead."
      ))
    } else {
      res_data = rename(res_data, any_of(labels)) 
    }
  }
```